keps/sig-node/3619-supplemental-groups-policy/kep.yaml

```diff
@@ -19,17 +19,21 @@ see-also: []
 replaces: []
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: alpha
+stage: beta
 
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.31"
+latest-milestone: "v1.33"
```

Comment:
We should not update "latest" to something iun the future.  The "latest" should stay 31

Reply:
Thanks, I changed back to 1.31 in https://github.com/kubernetes/enhancements/pull/4895/commits/69d37fea5c31fb026c12f457e176d16045d41ce1.